bokeh/util/string.py

```diff
@@ -1,33 +1,38 @@
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Copyright (c) 2012 - 2019, Anaconda, Inc., and Bokeh Contributors.
 # All rights reserved.
 #
 # The full license is in the file LICENSE.txt, distributed with this software.
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 ''' Functions useful for string manipulations or encoding.
 
 '''
 
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
 # Boilerplate
-#-----------------------------------------------------------------------------
+# -----------------------------------------------------------------------------
```

Comment:
I would actually ask that these spaces not be added. The convention in all other files is not to have a space before the `-` so changing here would introduce an inconsistency

Reply:
Spaces were autoadded by PyCharm, I have removed them, again.

Looking at the linter result now